man/repart.d.xml

```diff
@@ -576,6 +576,22 @@
         <varname>Verity=</varname>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>VerityDataBlockSizeBytes=</varname></term>
+
+        <listitem><para>Configures the data block size of the generated verity hash partition. Defaults to 4K for an
```

Comment:
"Defaults to the sector size if configured explicitly, or the underlying block device sector size, or 4K if systemd-repart is not operating on a block device"?

Reply:
Take a look now.